charts/jaeger/templates/collector-svc.yaml

```
@@ -41,6 +41,7 @@ spec:
   selector:
     {{- include "jaeger.selectorLabels" . | nindent 4 }}
     app.kubernetes.io/component: collector
+  clusterIP: {{ .Values.collector.service.clusterIP }}
```

```suggestion
  {{- if and (eq .Values.collector.service.type "ClusterIP") .Values.collector.service.clusterIP }}
  clusterIP: {{ .Values.collector.service.clusterIP }}
  {{- end }}
```

As to avoid `clusterIP: ` rendering with no value. Also have seen some bad behaviour when that is not handled: https://github.com/helm/helm/issues/7956

---

We could also check if `Values.clusterIP` is not `""`